### DISCUSSION TITLE
Is Faster Whisper good for languages such as Chinese and Malay?

### DISCUSSION THREAD

**Discussion by toanhn** [SYSTRAN/faster-whisper#1050] [Q&A]
Is Faster Whisper good for languages such as Chinese and Malay?

**Comment by Hikari-Tsai** [comment]
Yes. Multilingual Whisper models support both Chinese (`zh`) and Malay (`ms`), and faster-whisper preserves that capability. Do not use an `.en` model, because those checkpoints support English only.

If you know the language, specify it rather than relying on automatic detection:

```python
model = WhisperModel("large-v3", device="cuda", compute_type="float16")

segments, info = model.transcribe("chinese.wav", language="zh")
# or language="ms" for Malay
```

For Chinese, Whisper normally produces the writing system learned by the model; it is not a configurable Simplified/Traditional conversion system, so apply a dedicated conversion step if your output requirement is strict. For Malay, accuracy will depend strongly on code-switching, accent, recording quality, and domain vocabulary.

`large-v3` is the safest accuracy baseline; `turbo`, smaller models, and quantized compute trade some accuracy for speed or memory. Measure character error rate for Chinese and word error rate for Malay on representative labeled recordings before choosing a deployment model.
